src/store/modules/bot-commands/bot-commands-mutations.ts

```diff
@@ -0,0 +1,13 @@
+import { MutationTree } from 'vuex'
+import { BotCommand, BotCommandsState } from '@/store/modules/bot-commands/types.ts'
+
+export const mutations: MutationTree<BotCommandsState> = {
+  addCommand(state, botCommand: BotCommand): void {
+    const commandValue = botCommand.command.trim()
+    if (!state.commands[botCommand.partnerId]) {
```

Comment:
Can we set `botCommands = state.commands[botCommand.partnerId]` to make code simplier?

Reply:
```
addCommand(state, botCommand: BotCommand): void {
  const commandValue = botCommand.command.trim();
  let botCommands = state.commands[botCommand.partnerId];

  if (!botCommands) {
    state.commands[botCommand.partnerId] = [commandValue];
  } else if (!botCommands.includes(commandValue)) {
    botCommands.push(commandValue);
  }
}
```

Reply:
May be we also can init `state.commands[botCommand.partnerId] = []` somewhere on start and keep only

```
addCommand(state, botCommand: BotCommand): void {
  const commandValue = botCommand.command.trim();
  let botCommands = state.commands[botCommand.partnerId];

  if (!botCommands.includes(commandValue)) {
    botCommands.push(commandValue);
  }
}
```

Reply:
> May be we also can init `state.commands[botCommand.partnerId] = []` somewhere on start and keep only
> 
> ```
> addCommand(state, botCommand: BotCommand): void {
>   const commandValue = botCommand.command.trim();
>   let botCommands = state.commands[botCommand.partnerId];
> 
>   if (!botCommands.includes(commandValue)) {
>     botCommands.push(commandValue);
>   }
> }
> ```

It seems to me that this will be dangerous, because... There is a life cycle problem, namely at what point to call initialization? We can call initialization at start. But what about the new chat? What about updating the page? Whether the method will be called or not. One way or another, you will need to check that the array of commands already exists.
It seems to me inappropriate to increase the complexity of the life cycle in order to save one line of code.